orderer/common/msgprocessor/maintenancefilter.go

```diff
@@ -46,9 +46,7 @@ func NewMaintenanceFilter(support MaintenanceFilterSupport, bccsp bccsp.BCCSP) *
 		bccsp:                         bccsp,
 	}
 	mf.permittedTargetConsensusTypes["etcdraft"] = true
-	// Until we have a BFT consensus type, we use this for integration testing of consensus-type migration.
-	// Caution: proposing a config block with this type will cause panic.
-	mf.permittedTargetConsensusTypes["testing-only"] = true
+	mf.permittedTargetConsensusTypes["BFT"] = true
```

Comment:
This is the permitted target type. So you have to remove "etcdraft", because we only allow etcdraft -> BFT, but the other way around.

Reply:
In the `inspect` method, line 136, you to check that if the target is BFT, you can unmarshall the metadata to &smartbft.Options{}. Make sure all the field are recognized ;-) in unit test